1-exercises/B-setting-values/exercise2.js

```diff
@@ -16,6 +16,10 @@ let student = {
 */
 
 // write code here
+student[`attendance`] = 90;
+if (student['attendance'] >= 90 && student['examScore'] > 60) {
+  student['hasPassed'] = true;
+}
```

Comment:
You can implement this logic using an inline conditional, can you try doing that as well?